server/src/main.rs

```diff
@@ -183,48 +183,54 @@ async fn handle_socket(socket: WebSocket, state: Arc<StdMutex<State>>) {
                 let _ = local_sender.send(Response::ServerTime(SystemTime::now().duration_since(std::time::UNIX_EPOCH).unwrap().as_millis() as u64)).await;
             },
             Request::JoinRoom(room_name, player_name, x, y, z) => {
-                log::info!("Player {player_id:?} ({player_name}) joins room {room_name:?}");
+                if room_name.len() > 128 {
+                    log::error!("Room name is greater than 128 chars.");
```

Comment:
It's a `warn` not an `error`. Error usually indicates a problem of the server's implementation. Also please add the player-id, player-name and room-name to the message for better debugging.